docs/projects/led-guide.md

```diff
@@ -0,0 +1,524 @@
+# Creating your first node
+
+## Repo Structure
+
+At first the repository will seem a bit intimidating! There's plenty
+going on here, but for the purposes of this guide, you don't need to
+worry about most of it.
+
+As a prequiste, make sure you've read through Jacob's
+[getting started guide](https://github.com/CooperUnion/selfdrive#getting-started)
+on setting up the repository.
```

Comment:
maybe have them create their own branch first so that not everyone is working off dev?